aggregate/housing_security/homevalue_median.py

```diff
@@ -0,0 +1,49 @@
+"""this chops up into six indicators they are """
+from typing import final
+import pandas as pd
+from aggregate.clean_aggregated import rename_col
+from utils.PUMA_helpers import clean_PUMAs, borough_name_mapper, get_all_boroughs, get_all_NYC_PUMAs
+from utils.dcp_population_excel_helpers import race_suffix_mapper, stat_suffix_mapper_md
+from internal_review.set_internal_review_file import set_internal_review_files
+from aggregate.load_aggregated import load_clean_pop_data
+
+year_mapper = {"12": "0812", "19": "1519"}
+
+def homevalue_median(geography: str, write_to_internal_review=False) -> pd.DataFrame:
+
+    name_mapper = {
+        "MdVl": "homevalue_median"
+    }
+
+    ind_name_str = "|".join([k for k in name_mapper.keys()])
```

Comment:
curious to see why you added this step but still haven't run it on my machine, just different than what I did